tests/RLS/TraitManagerTest.php

```diff
@@ -262,10 +264,10 @@
 
     expect(fn () => DB::statement("INSERT INTO comments (text, post_id) VALUES ('third comment', {$post1->id})"))
         ->toThrow(QueryException::class);
-})->with([
-    true,
-    false
-]);
+})->with(
+    [true, false],
+    [true, false],
+);
```

Comment:
```suggestion
})->with(
    [true, false],
    [false, true],
);
```
or
```suggestion
})->with([true, false])->with([true, false]);
```

Reply:
Hmm or is the syntax that it builds a matrix from those arrays? I see in another dataset it's used that way.

Probably change these to use two `with()` calls to make that more clear.